src/containers/location/providers/ReferralAgencyHeaderContainer.js

```diff
@@ -0,0 +1,148 @@
+// @flow
+
+import React from 'react';
+
+import styled from 'styled-components';
+import { faChevronLeft } from '@fortawesome/pro-light-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { Colors } from 'lattice-ui-kit';
+import { useSelector, useDispatch } from 'react-redux';
+
+import { selectReferralAgency } from '../LocationsActions';
+import { ContentOuterWrapper, ContentWrapper } from '../../../components/layout';
+import { HEADER_HEIGHT, HEIGHTS } from '../../../core/style/Sizes';
+import { getTextFnFromState } from '../../../utils/AppUtils';
+import { renderFacilityName } from '../../../utils/DataUtils';
+import { LABELS } from '../../../utils/constants/Labels';
+import { PROVIDERS, STATE } from '../../../utils/constants/StateConstants';
+
+const { LOCATIONS } = STATE;
+
+const { NEUTRAL, PURPLE } = Colors;
+
+const StyledContentOuterWrapper = styled(ContentOuterWrapper)`
+  position: fixed;
+  top: ${HEADER_HEIGHT}px;
+  z-index: 1;
+`;
+
+const StyledContentWrapper = styled(ContentWrapper)`
+  background-color: white;
+  position: relative;
+
+  @media only screen and (min-height: ${HEIGHTS[0]}px) {
+    padding: 10px 25px;
+  }
+
+  @media only screen and (min-height: ${HEIGHTS[1]}px) {
+    padding: 25px;
+  }
+`;
```

Comment:
I feel like I've seen similar (or perhaps exact) components in other files. It would be nice to avoid duplicating the same functionality in many places.